cfgov/unprocessed/js/config/error-messages-config.js

```diff
@@ -7,13 +7,21 @@
 'use strict';
 
 var ERROR_MESSAGES = {
-  CHECKBOX_ERRORS: {
-    required: 'Please select at least one of the "%s" options.'
+  CHECKBOX : {
+    required : 'Please select at least %s of the options.'
```

Comment:
Hm, and it can be modified even though ERROR_MESSAGES is exported with `Object.seal( )`? Interesting, maybe object seal only works on the direct object it wraps. Technically a constant wouldn't be modified at any level of its structure. Not a big deal now, but this may require a change in futureland where `var ERROR_MESSAGES` becomes `const ERROR_MESSAGES`.